lib/xnvme_be_posix_async_aio.c

```diff
@@ -81,7 +81,7 @@ _posix_async_aio_poke(struct xnvme_queue *q, uint32_t max)
 	req = TAILQ_FIRST(&queue->reqs_outstanding);
 	assert(req != NULL);
 
-	while (req != NULL && completed < max) {
+	while (req != NULL && req->ctx != NULL && completed < max) {
```

Comment:
scan-build is a bit annoying here since this should never happen as entries entering ``reqs_outstanding`` always have a ``ctx``.  Hmm...